src/data-pipeline/lambda/emr-serverless-app/index.ts

```diff
@@ -77,7 +77,7 @@ async function createEMRServerlessApp(props: ResourcePropertiesType): Promise<st
     name: props.name,
     releaseLabel: props.version,
     type: 'SPARK',
-    architecture: Architecture.X86_64,
+    architecture: 'X86_64',
```

Comment:
add/update test case for it

Reply:
already have UT covered the case.  

https://github.com/awslabs/clickstream-analytics-on-aws/blob/main/test/data-pipeline/emr-serverless-app.test.ts#L80

There was no issue in UT. it cased by compile-time, from `ts` to `js` by CDK's `NodejsFunction`.

Reply:
What’s wrong when using NodejsFunction which uses esbuild to build the optimized js?

If it’s an issue from CDK, please report it to community too.

Reply:
raised issue to CDK: https://github.com/aws/aws-cdk/issues/26681